### PR TITLE
Update about si-verb formation

### PR DESCRIPTION
**Pull request by vawm**
Added interjections + slang

### CODE REVIEW

File: en/syntax.tex [hunk 1]
```diff
@@ -27,11 +27,12 @@ \subsection{Transitivity} Na'vi marks the subject of transitive and
...
 \subsubsection{} Many compound verbs are created by pairing an
-uninflected noun or adjective with the verb stem \N{si} \E{to do, to
-make}, which is only used in these compounds, \N{irayo si} \E{to
-thank}, \N{kavuk si} \E{to betray}.  These verbs are always
-intransitive and use the dative for any object
-(\horenref{syn:case:dative-si}).
+uninflected noun or adjective, or occasionally an interjection,
+with the verb stem \N{si} \E{to do, to make}, which is only used 
+in these compounds, \N{irayo si} \E{to thank}, \N{kavuk si} \E{to betray}.
+Proper names may also be used in very casual conversations as slang.
```
Comment (Willem3141) [inline en/syntax.tex:33]: It may make more sense to put this in section 7.3.3 on the colloquial register?

File: en/syntax.tex [hunk 2]
```diff
@@ -27,11 +27,12 @@ \subsection{Transitivity} Na'vi marks the subject of transitive and
...
+These verbs are always intransitive and use the dative for any object
+(\horenref{syn:case:dative-si}
```
Comment (Willem3141) [inline en/syntax.tex:35]: Seems like a parenthesis and full stop were dropped here

**Review by Willem3141 — CHANGES_REQUESTED** [review]

File: en/syntax.tex (hunk 1 above)
Comment (tirea) [reply en/syntax.tex:33]: Yeah, that makes sense. Somehow I forgot that there was a section on that

Comment (vawm) [reply en/syntax.tex:33]: You're not alone hrh

File: en/syntax.tex (hunk 2 above)
Comment (tirea) [reply en/syntax.tex:35]: Fixed this in latest commit

**Pull request merged**